### ISSUE TITLE
Percentile throws NULLPOINTEREXCEPTION when not grouping by a column

### ISSUE THREAD

**Issue by bputt** [crate/crate#4353]
Crate version: 0.56.3

This query throws a NULLPOINTEREXCEPTION
```
select percentile(field, 0.95) from table
```

This query does not
```
select percentile(field, 0.95), someOtherField from table group by someOtherField
```

**Comment by bogensberger** [comment]
Hi @bputt, 

thanks for reporting this issue. Could you please provide me some more information? What kind of column is `field` (you can provide me the output of `SHOW CREATE TABLE table` if you want) ?
Do you have any data in it? Is the table empty?

**Comment by bputt** [comment]
Hello @bogensberger,

The column type is ```LONG```

There is data in the table

**Comment by bputt** [comment]
@bogensberger 

Additional info:
This also occurs on ```INT``` fields as well, but here's the error

```
SQLActionException: INTERNAL_SERVER_ERROR 5000 NullPointerException in io.crate.types.DataTypes.fromStream(DataTypes.java:159)
```

**Comment by kovrus** [comment]
Hi @bputt, the fix will be released soon in `0.56.4` and `0.57.1`.

**Comment by mfussenegger** [comment]
Closing this, 0.56.4 and 0.57.1/2 have been released already.
